clang/lib/DPCT/DPCT.cpp

```diff
@@ -843,6 +857,19 @@ int runDPCT(int argc, const char **argv) {
 
   std::vector<std::string> SourcePathList;
   if (QueryAPIMapping.getNumOccurrences()) {
+    if (QueryAPIMapping.getNumOccurrences() > 1) {
+      llvm::outs()
+          << "Warning: Option --query-api-mapping is specified multi times, "
+             "only the last one is used, all other are ignored.\n";
+    }
+    if (hasOptConflictWithQuery(argc, argv)) {
+      llvm::outs()
+          << "Warning: For API mapping query function triggered by option "
+             "--query-api-mapping, only \"--extra-arg\" and "
+             "\"--cuda-include-path\" option can be set.\n";
+      ShowStatus(MigrationErrorConflictOptions);
```

Comment:
For API mapping query, only option --extra-arg and --cuda-include-path can be used together with option --query-api-mapping.